dataactvalidator/scripts/loaderUtils.py

```diff
@@ -91,6 +91,6 @@ def clean_data(data, model, field_map, field_options):
 def format_date(value):
     """ Format date from 'MMM dd, yyyy' to 'yyyymmdd' """
 
-    formatted_value = pd.to_datetime(value, format="%b, %d %Y")
+    formatted_value = pd.to_datetime(value, format="%b %d,%Y")
```

Comment:
Is this date format something you changed? Something from a merge? Why did it need to change?

Reply:
Correct. This was necessary for the CFDA loader.